pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
```

Comment:
Suggestion: `will := william.New(...)`  ;)

We're missing the  package import in the snippet, in practice I'd expect it to be something like:

```go
import william github.com/topfreegames/Will.IAM/client

will := william.New(...)
```